java/arcs/core/host/BUILD

```diff
@@ -0,0 +1,14 @@
+load("//third_party/java/arcs/build_defs:build_defs.bzl", "arcs_kt_jvm_library")
+
+licenses(["notice"])
+
+package(default_visibility = ["//visibility:public"])
+
+arcs_kt_jvm_library(
```

Comment:
Can you use `kt_jvm_and_js_library` here instead? All `core` code should be buildable on both.